client/src/components/UserCard/UserCard.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Grid,
+  Paper,
+  Typography,
+  Box,
+  Card,
+  TextField,
+  Button,
+  CardActions,
+  CardContent,
+  Divider,
+  CardMedia,
+  CardActionArea,
+  Avatar,
+  Input,
+} from '@material-ui/core';
+import { Rating } from '@material-ui/lab';
+
+import { Link } from 'react-router-dom';
+
+import useStyles from './useStyles';
+const profileImg = '775db5e79c5294846949f1f55059b53317f51e30.png';
+const placeholder = {
+  rating: 5,
+  price: 15,
+  location: 'Toronto, Ontario',
+};
+const defaultDescription = 'New pet sitter!';
+
+export default function UserCard(props: any): JSX.Element {
+  const classes = useStyles();
+  const { firstName, lastName, user, description, address } = props.profile;
+  return (
+    <Grid item key={user}>
+      <Card style={{ height: '100%' }} variant="outlined" key={user}>
```

Comment:
Same here about the `key` prop. Also, please reserve the use of inline styles for local development/testing - we should clean them up before shipping the code (e.g. making a PR) 👍🏼

Reply:
Noted, thank you!